jOOQ/src/main/java/org/jooq/impl/DDL.java

```diff
@@ -78,6 +78,10 @@
     }
 
     final Queries queries(Table<?> table) {
```

Comment:
I guess, the single table method can be removed from the internal API to remove redundancy a little bit.